content/working_with/console/widgets/filters.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: bt_wiki
+title: Filters
+category: Widgets
+draft: false
+---
+
+Displays the list of defined filters.
+The table has 4 columns:
+
+* **Filter name** - filter ID
+* **Creator** - filter creator
+* **Created** - filter creation date and time
+* Actions column
+
+![filters]( /images/ui/widgets/filters.png )
+
+Actions column contains following action icons:
+
+* **Edit filter** opening filter rules edit modal (see [defining filter rules]({{< relref "#defining-filter-rules" >}}) for details on rule specifying)
+* **Clone filter** opening filter clone modal allowing for creating a possibly modified version of selected filter (see [defining filter rules]({{< relref "#defining-filter-rules" >}}) for details on rule specifying)
+* **Delete filter** allowing for removal of selected filter (see note below)
+
+Deleting filter which is used as a default filter in Deployments View widget is not possible.
+When trying to delete such a filter a modal shows up describing where (on which page and in which widget) and by whom (by which user) the filter is used.
+
+![filters delete]( /images/ui/widgets/filters-delete.png ) 
+
+
+Clicking the `Add` button above the table opens the filter add modal allowing for creation of a new filter by specifying filter ID and filter rules.
+
+## Defining filter rules
+
+Add, edit and clone operation modals share a common component for defining filter rules.
+
+![filters rules]( /images/ui/widgets/filters-rules.png ) 
```

Comment:
The styling for label rule will change probably once I implement RD-2007.

Reply:
Looks like the only important change will be the space between key and value dropdowns. If so I could manually edit the image and introduce spacing so the image reflected future change and the PR was not blocked.

Reply:
In case of this screenshot, it seems the only change.

Alternatively, you can also generate it using the code from https://github.com/cloudify-cosmo/cloudify-stage/pull/1315 .